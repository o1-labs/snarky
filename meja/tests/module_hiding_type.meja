module A = {
  type t;
};

module B = {
  module A = {
    type u;
  };
};

open B;

let f = fun (x : A.u) => { x; };

/* Expect an error, A should be out of scope. */
let g = fun (x : A.t) => { x; };
