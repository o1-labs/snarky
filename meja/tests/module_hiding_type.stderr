File "tests/module_hiding_type.meja", line 16, characters 17-20:
Error: Unbound type A.t.
